package.json

```diff
@@ -36,6 +37,7 @@
     "google-oauth-jwt": "0.1.7",
     "googleapis": "0.8.0",
     "lodash": "2.4.1",
+    "meetup-api": "^1.4.9",
```

Comment:
Since we already had these kind of problems in this repo couple of times already, I would also suggest to keep them strict for reproducable builds.